### ISSUE TITLE
Default value for i.seasons in memmodel

### ISSUE THREAD

**Issue by jrh-dev** [lozalojo/mem#2]
Consider removing the default value of 10  for i.seasons in memmodel function. May inadvertently lead to the function including a week identifier column in calculations.

**Comment by lozalojo** [comment]
> Consider removing the default value of 10 for i.seasons in memmodel function. May inadvertently lead to the function including a week identifier column in calculations.

Not sure if I understood right, do you mind providing an example?

**Comment by jrh-dev** [comment]
Based on the mem::memmodel documentation below;

"The first column should contain the names of the weeks. When the season contains two different calendar years, the week will go from 40th of the first year to 52nd, and then from 1st to 20th. When the season contains one year, the weeks will go from 1st to 52nd.

Note: If there is no column with week names, the application will name the weeks numbering from 1 to the number of rows."

I had been passing a data.frame as per the example below. As per the 3 examples the 'week_id' column is included in the model when the i.seasons argument is left as default and the number of seasons is less than 10. This may be intended behaviour but I didn't feel the documentation suggested this to be the case. 

```
example_data <- data.frame(
  c(40:52,1:20),
  abs(rnorm(33)) * 10,
  abs(rnorm(33)) * 10,
  abs(rnorm(33)) * 10
)

names(example_data) <- c("week_id", "2016/17", "2017/18", "2018/19")

#example 1
mem::memmodel(example_data)

#example 2
mem::memmodel(example_data[2:4])

#example 3
mem::memmodel(example_data, i.seasons = 3)
```

**Comment by lozalojo** [comment]
Yes, seems like something strange is happening there. Honestly most column-wise checks were implemented on memapp, and not mem, so maybe I forgot to add some checks to the input data.

**Comment by lozalojo** [comment]
Fixed. In fact, mem was never designed to include week names in the dataset as stated in the documentation, which I probably copied from memapp documentation.

Anyway, I've included part of the input data check routines of memapp and now memmodel accepts week names at the first column.

All following examples produce the same output:

```summary(temp1<-mem::memmodel(example_data))
summary(temp2<-mem::memmodel(example_data[2:4]))
summary(temp3<-mem::memmodel(example_data, i.seasons = 3))
summary(temp4<-mem::memmodel(example_data[2:4], i.seasons = 3))
```

Finally, If you are not using memapp, I'd recommend you using it; it is a gui for mem:

https://github.com/lozalojo/memapp
